block-storage-attacher/tests/e2e/e2e-tests/portworx_pvc.yaml

```diff
@@ -0,0 +1,12 @@
+kind: PersistentVolumeClaim
```

Comment:
Is this PVC used anywhere? I didnt find any place in e2e where its used.

Reply:
Earlier I was creating the PVC using the file . Later changed the logic I will remove it

Reply:
Done

Reply:
You said Done but the file is not removed from PR.